.env

```
@@ -0,0 +1 @@
+API_BASE_URL=https://speako.site
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)